voice/src/main/kotlin/gateway/Command.kt

```diff
@@ -72,6 +76,12 @@ public data class SendSpeaking(
     val ssrc: UInt
 ) : Command()
 
+@KordVoice
+@Serializable
+public data class MediaSinkWants(
+    val any: Int
```

Comment:
can you add a comment to describe what this does? `any` isn't a very descriptive name...